CMakeLists.txt

```diff
@@ -83,9 +83,15 @@ include_directories(
 
 link_directories(/usr/local/lib)
 
-add_definitions(
-  -DSHARE_PATH="${SHARE_PATH}"
-)
+if(SHARE_PATH)
+  add_definitions(-DSHARE_PATH="${SHARE_PATH}")
+elseif(SHARE_INSTALL_PREFIX)
+  add_definitions(-DSHARE_PATH="${SHARE_INSTALL_PREFIX}/baresip")
+else()
+  add_definitions(
+    -DSHARE_PATH="${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DATADIR}/baresip"
```

Comment:
```suggestion
    -DSHARE_PATH="${CMAKE_INSTALL_FULL_DATADIR}/baresip"
```

Reply:
Ha, `CMAKE_INSTALL_FULL_DATADIR` was actually what I was looking for instead of `SHARE_INSTALL_PREFIX`; that simplifies the code even.